common/config.go

```diff
@@ -147,4 +147,11 @@ const (
 	OptCNIConflistScenario = "cni-conflist-scenario"
 	// OptCNIConflistScenarioAlias "shorthand" for the cni conflist scenairo, see above
 	OptCNIConflistScenarioAlias = "cniconflistscenario"
+
+	// OptWatcherPath
+	OptWatcherPath      = "watcher-path"
+	OptWatcherPathAlias = "wp"
+	// OptDeleteDirectory
+	OptDeleteDirectory      = "delete-directory"
+	OptDeleteDirectoryAlias = "deletedir"
```

Comment:
we don't need flags for this, the options being in the config file is sufficient